code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1336,4 +1336,151 @@
+
+	if(prob(10)) // Start of base probability effects
+		var/tox_roll = pick(1,-1) // Rolls for damage or healing
+		if(tox_roll == 1)
+			M.emote("moan")
+		if(tox_roll == -1)
+			M.emote("twitch")
+		if(prob(90))
+			M.adjustToxLoss(tox_roll*REM, 0)
+		M.Jitter(jitter)
+	if(prob(1)) // Last set of non OD probability effects
+		M.Dizzy(jitter)
+		M.slurring = slur // End of base probability effects
+	if(heal_roll < 0) // Healing payload after calculations
+		if (M.stat == DEAD) // Non OD , if dead , 30% chance to discard healing payload
+			if(prob(70))
+				M.adjustFireLoss(heal_roll*REM, FALSE, FALSE, BODYPART_ORGANIC)
+				M.adjustBruteLoss(heal_roll*REM, 0)
+		else
+			M.adjustFireLoss(heal_roll*REM, 0) 
+			M.adjustBruteLoss(heal_roll*REM, 0) 
+	..()
+	. = 1
+
+/datum/reagent/drug/burnmix/overdose_start(mob/living/M)
+	to_chat(M, "<span class='userdanger'>You feel the BurnMix increase in potency.</span>")
+
+
+/datum/reagent/medicine/burnmix/overdose_process(mob/living/M)
+	var/heal_roll = pick(-0.2,-0.4)
+	var/slur = 20
+	var/jitter = 2
+	if(M.health <= M.crit_threshold)
+		heal_roll = heal_roll + 0.2 // Burnmix in overdose will heal inside of crit without epipen or perfluorodecalin. Being in crit means a roll for either no base healing or half reduction to base heal.
+	if(heal_roll > 0 && prob(10))
+		heal_roll = 0 // Burnmix has a chance not to basic heal at all
+	if(holder.has_reagent(/datum/reagent/medicine/epinephrine))
+		heal_roll = heal_roll - 0.35
+		jitter = jitter + 15
+		slur = slur + 25
+		if(prob(25))
+			M.losebreath++
+			M.adjustStaminaLoss(2.5*REM, 0)
+			if(prob(80))
+				M.adjustToxLoss(0.5*REM, 0)
+				M.adjustToxLoss(pick(0,-0.25), 0) // gives a chance to negate half epinephrine tox damage
+
+	if(holder.has_reagent(/datum/reagent/medicine/perfluorodecalin))
+		heal_roll = heal_roll - 0.4
+		jitter = jitter + 15
+		slur = slur + 25
+		if(prob(25))
+			M.adjustToxLoss(0.5*REM, 0)
+			M.adjustToxLoss(pick(0,-0.25), 0) // gives a chance to negate half perfluorodecalin tox damage
+
+	if(holder.has_reagent(holder.has_reagent(/datum/reagent/medicine/epinephrine) && /datum/reagent/medicine/perfluorodecalin)) // start IF both drugs present
+		if(prob(0.5)) // Should be 1 in 200 chance of a 1 in 3 chance to roll for faint if both epinephrine and perfluorodecalin are present
+			M.emote((pick("scream","faint","gasp")))
+		if(prob(60)) // Original edit did too much brain damage. Moved to both drug effects. Was default OD effect
+			M.adjustOrganLoss(ORGAN_SLOT_BRAIN,pick(1,-1))
+		if(prob(30))
+			slur = slur + 25
+		if(prob(30))
+			jitter = 50
+		if(prob(30))
+			M.drowsyness += 0.5
+			if(prob(2))
+				to_chat(M, "<span class ='notice'>Your fingers spasm!</span>")
+				M.drop_all_held_items() // end IF both drugs present
+
+	if(prob(40)) // start last set of OD probability effects
+		M.emote(pick("twitch","drool","moan","giggle","spin"))
+		if(prob(60))
+			M.adjustOrganLoss(ORGAN_SLOT_BRAIN, pick(0,1,2,2.5,-1,-2))
```

Comment:
```suggestion
			M.adjustOrganLoss(ORGAN_SLOT_BRAIN, rand(-2,3))
```

Reply:
Hello Alex,
How can we ensure that the 2.5 makes in it to the random selection? The difference of the 0.5 being an option in testing was about 8 units of brain damage after consuming 60 units of BurnMix which was very substantial in my opinion. The 2.5 increment is there on purpose for balancing reasons.

Reply:
I will modify this if it created an issue otherwise I am going to go ahead and resolve this inquiry as it is intentional with the 2.5 being placed in the choices for balance reasons.
Please re-open this is there is further contention as I am open to collaborating.